### PR TITLE
Pin gems for Ruby <2.2 (1.16-stable)

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: bundler.d/test.rb [hunk 1]
```diff
@@ -16,4 +14,16 @@
...
     gem 'webmock'
   end
+
+  if RUBY_VERSION < '2.2.2'
+    gem 'rack-test', '~> 0.7.0'
+  else
+    gem 'rack-test'
+  end
+
+  if RUBY_VERSION < '2.2'
```
Comment (dmitri-d) [inline bundler.d/test.rb:24]: This duplicates the version spec in development.rb; this one can be removed, no?

Comment (mmoll) [reply bundler.d/test.rb:24]: rdoc was part of both files before this change also, I guess when one does exclude one of the two bundler groups, the problem still could happen when it's not in both.

Comment (dmitri-d) [reply bundler.d/test.rb:24]: Nods (on both accounts). Test probably won't be used without development though, but I'll leave it up to you.

Comment (dmitri-d) [reply bundler.d/test.rb:24]: Looks good otherwise!

Comment (mmoll) [reply bundler.d/test.rb:24]: I'll leave it as-is here, then. If this should get tackled, it would need to happen in develop anyway. :hammer_and_wrench:

**Comment by dmitri-d** [comment]
Merged, thanks @mmoll

**Pull request merged**